### PR TITLE
Added format-amount component

### PR DESCRIPTION
**Pull request by razvantomegea**

### CODE REVIEW

File: src/components/format-amount/format-amount.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  shadow: true
+})
+export class FormatAmount {
+  @Prop() class?: string;
+  @Prop() decimals?: number = DECIMALS;
+  @Prop() digits?: number = DIGITS;
+  @Prop() egldLabel?: string;
+  @Prop() showLabel?: boolean = true;
+  @Prop() showLastNonZeroDecimal?: boolean = false;
+  @Prop() styles?: { [key: string]: string } = {};
```
Comment (arhtudormorar) [inline src/components/format-amount/format-amount.tsx:18]: is this needed?

Comment (razvantomegea) [reply src/components/format-amount/format-amount.tsx:18]: Yes, it is used

File: src/components/format-amount/format-amount.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+import BigNumber from 'bignumber.js';
+import { formatAmount } from '@multiversx/sdk-dapp-utils/out/helpers';
+import { DECIMALS, DIGITS, ZERO } from '@multiversx/sdk-dapp-utils/out/constants';
+
+@Component({
+  tag: 'format-amount',
+  styleUrl: 'format-amount.css',
+  shadow: true
+})
+export class FormatAmount {
```
Comment (arhtudormorar) [inline src/components/format-amount/format-amount.tsx:11]: add tests like in sdk-dapp see ledger tests

File: package.json [hunk 3]
```diff
@@ -47,20 +51,22 @@
     "generate": "stencil generate"
   },
   "dependencies": {
-    "qrcode": "1.5.4",
-    "classnames": ">= 2.5.1",
+    "@fortawesome/fontawesome-svg-core": ">= 6.7.2",
     "@fortawesome/free-solid-svg-icons": ">= 6.7.2",
-    "@fortawesome/fontawesome-svg-core": ">= 6.7.2"
+    "@multiversx/sdk-dapp-utils": "1.0.2",
```
Comment (arhtudormorar) [inline package.json:56]: `>= 1.0.2`

File: package.json [hunk 4]
```diff
@@ -47,20 +51,22 @@
...
+    "bignumber.js": "9.1.2",
```
Comment (arhtudormorar) [inline package.json:57]: `>=`

**Pull request merged**